File: src/main/java/org/example/collection/Country.java
package org.example.collection;

import java.util.LinkedList;

public enum Country{
    RUSSIA,
    UNITED_KINGDOM,
    GERMANY,
    VATICAN;
    public static String getValue() {
        LinkedList<String> EnumValues = new LinkedList<>();
        for (Country country : Country.values()){
            EnumValues.add(country.toString());
        }
        return String.join(", ", EnumValues);
    }
}

File: common/src/main/kotlin/collection/Country.kt
package collection

import java.io.Serializable
import java.util.Arrays
import java.util.stream.Collectors

enum class Country : Serializable {
    RUSSIA,
    UNITED_KINGDOM,
    GERMANY,
    VATICAN,
    ;

    companion object {
        val value: String
            get() =
                Arrays.stream(entries.toTypedArray())
                    .map { obj: Country -> obj.name }
                    .collect(Collectors.joining(", "))
    }
}
